### PR TITLE
Silver Searcher updated recipe

### PR DESCRIPTION
**Pull request by miqlas**
Tested, damned fast. :)

### CODE REVIEW

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe [hunk 1]
```diff
@@ -0,0 +1,73 @@
+SUMMARY="A code-searching tool similar to ack, but faster"
+DESCRIPTION="A code-searching tool similar to ack, but faster.
```
Comment (diversys) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Duplicates summary

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	the_silver_searcher$secondaryArchSuffix = $portVersion
+	cmd:ag$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:liblzma${secondaryArchSuffix}
```
Comment (diversys) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Lose braces everywhere except haiku${secondaryArchSuffix}_devel

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+	autoheader
+	automake --add-missing
+
+	#Need to override the pkgconfig variables
+	LZMA_CFLAGS="-I$includeDir" \
+	LZMA_LIBS="-L$libDir -llzma" \
+	PCRE_CFLAGS="-I$includeDir" \
+	PCRE_LIBS="-L$libDir -lpcre" \
+	runConfigure --omit-dirs binDir ./configure \
+	--bindir=$commandBinDir
```
Comment (diversys) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Perhaps move overrides after runConfigure --omit-dirs binDir to add identation?

**Comment by diversys** [comment]
According to Travis there is a trailing whitespace in the recipe, but I don't see where...

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* It ignores file patterns from your .gitignore and .hgignore.
+* If there are files in your source repo you don't want to search, just add \
+their patterns to a .agignore file.
+* Ag uses Pthreads to take advantage of multiple CPU cores and search files in \
+parallel."
+HOMEPAGE="https://github.com/ggreer/the_silver_searcher/"
+COPYRIGHT="2013-2016 Geoff Greer"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="git+https://github.com/ggreer/the_silver_searcher.git#0.32.0"
```
Comment (fbrosson) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: I think you can replace
```
SOURCE_URI="git+https://github.com/ggreer/the_silver_searcher.git#0.32.0"
```
by
```
SOURCE_URI="https://github.com/ggreer/the_silver_searcher/archive/$portVersion.tar.gz"
CHECKSUM_SHA256="c5b208572e5cfc8a3cf366e8eb337b0c673c2ffa90c1ad90dfdcbe78251ba4cc"
```

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe (hunk 3 above)
Comment (fbrosson) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Hmm, I think this is OK as it is.

Well, I'd just add an extra tab before **`--bindir=$commandBinDir`**
**_or_** join the last two lines into a single one:
```
	runConfigure --omit-dirs binDir ./configure --bindir=$commandBinDir
```

**Comment by TwoFX** [comment]
Please change the commit message (using `git commit --amend`) to something a little more descriptive (e.g. "the_silver_surfer: update to version 0.32.0")

File: sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe [hunk 5]
```diff
@@ -0,0 +1,75 @@
+SUMMARY="A code-searching tool similar to ack, but faster"
+DESCRIPTION="A  tool for searching code. It started off as a clone of Ack, but their \
```
Comment (waddlesplash) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Extra space.

Comment (miqlas) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Where? My cleaner script cannot find anything...

Comment (miqlas) [inline sys-apps/the_silver_searcher/the_silver_searcher-0.32.0.recipe]: Ok, i found it.

**Pull request merged**